opt4j-optimizers/src/main/java/org/opt4j/optimizers/ea/aeseh/EpsilonAdaption.java

```diff
@@ -0,0 +1,51 @@
+package org.opt4j.optimizers.ea.aeseh;
+
+import com.google.inject.ImplementedBy;
+
+/**
+ * The {@link EpsilonAdaption} manages the adaptation of the ε-value used for
+ * survivor selection by {@link AeSeHSelector}.
+ * 
+ * @author Fedor Smirnov
+ *
+ */
+@ImplementedBy(DefaultEpsilonAdaptation.class)
+public interface EpsilonAdaption {
```

Comment:
Is it likely that we get another EpsilonAdaption? To me, this also looks extremely AeSeH specific. If not, we could remove the interface in favor for just a single class.